myblog/models.py

```diff
@@ -8,4 +8,5 @@ class Post(models.Model):
     summary = ckfields.RichTextField()
     content = ckupload_fields.RichTextUploadingField()
     author = models.ForeignKey(User, on_delete=models.PROTECT)
-    created_at = models.DateField(auto_now_add=True)
\ No newline at end of file
+    created_at = models.DateTimeField(auto_now_add=True)
+    last_updated = models.DateTimeField(auto_now=True)
```

Comment:
Aqui talvez seja frescura da minha parte com nome de campo/variavel, mas o que acha de
```suggestion
    updated_at = models.DateTimeField(auto_now=True)
```
pra ficar consistente com o campo de cima?

Reply:
Boa!